packages/bootstrap/extra/dcos_internal_utils/exhibitor.py

```diff
@@ -109,8 +109,9 @@ def wait(master_count_filename):
         "Serving hosts: `%s`, leader: `%s`", ','.join(serving), ','.join(leaders))
 
     if len(serving) != cluster_size or len(leaders) != 1:
-        msg_fmt = 'Expected {} servers and 1 leader, got {} servers and {} leaders'
-        raise Exception(msg_fmt.format(cluster_size, len(serving), len(leaders)))
+        msg_fmt = 'Expected {} servers and 1 leader, got {} servers and {} leaders\n'
```

Comment:
Why there is a `'\n'` at the end of msg string?

Reply:
Ah, good catch! That was added during a transitional point when I logged the message to stderr. I will change it back.